charts/incubator/ps3netsrv/Chart.yaml

```
@@ -0,0 +1,38 @@
+annotations:
+  max_scale_version: 24.04.1
+  min_scale_version: 24.04.0
+  truecharts.org/SCALE-support: "true"
+  truecharts.org/category: games 
+  truecharts.org/max_helm_version: "3.15"
+  truecharts.org/min_helm_version: "3.11"
+  truecharts.org/train: incubator
+apiVersion: v2
+appVersion: 1.29.1
+dependencies:
+  - name: common
+    version: 23.0.11
+    repository: oci://tccr.io/truecharts
+    condition: ""
+    alias: ""
+    tags: []
+    import-values: []
+deprecated: false
+description: Allows Multiman or Webman to stream and play game backups over the network.
+home: https://truecharts.org/charts/incubator/ps3netsrv
+icon: https://truecharts.org/img/hotlink-ok/chart-icons/ps3netsrv.webp
+keywords:
+  - ps3netsrv
+  - Game servers
+  - Network-Other
+kubeVersion: '>=1.24.0-0'
+maintainers:
+  - name: TrueCharts
+    email: info@truecharts.org
+    url: https://truecharts.org
+name: ps3netsrv
+sources:
+  - https://github.com/aldostools/webMAN-MOD/tree/master/_Projects_/ps3netsrv
+  - https://github.com/truecharts/charts/tree/master/charts/incubator/ps3netsrv
+  - https://hub.docker.com/r/shawly/ps3netsrv
+type: application
+version: 1.0.9
```

```suggestion
version: 0.0.1
```
its an initial add of the app. So i would start with a lower version